libr/core/cbin.c

```diff
@@ -1223,13 +1220,12 @@ R_API int r_core_pdb_info(RCore *core, const char *file, ut64 baddr, int mode) {
 	}
 
 	pdb.print_types (&pdb, mode);
+	// I have to keep this unless I want to pass PJ around all the functions
 	if (mode == 'j') {
```

Comment:
this should be using the PJ api, but out of the scope of this PR

Reply:
Not an issue really, I can add it, I refactored each of the printing function to use PJ already